dao-modules/src/governance_voting/voting.rs

```diff
@@ -0,0 +1,112 @@
+use casper_dao_utils::Address;
+use casper_types::{
+    bytesrepr::{FromBytes, ToBytes},
+    CLType, CLTyped, RuntimeArgs, U256,
+};
+
+pub type VotingId = U256;
+
+#[derive(Debug, Default, Clone)]
+pub struct Voting {
+    pub voting_id: VotingId,
+    pub completed: bool,
+    pub stake_in_favor: U256,
+    pub stake_against: U256,
+    pub finish_time: U256,
```

Comment:
Shouldn't we use `u64` as a time unit. Casper uses u64, so sooner or later we add U256 to u64

Reply:
Changed all time variables to u64. Simplified things a bit.